examples/bridge-app/bridge-common/bridge-app.zap

```diff
@@ -1528,7 +1528,16 @@
           "define": "GENERAL_DIAGNOSTICS_CLUSTER",
           "side": "client",
           "enabled": 0,
-          "commands": [],
+          "commands": [
+            {
+              "name": "TestEventTrigger",
+              "code": 0,
+              "mfgCode": null,
+              "source": "client",
+              "incoming": 1,
```

Comment:
No, this does not make any sense.  This is not an "incoming" command for the _client_ side of the cluster!